### PR TITLE
fix(35/*) fix all navigation 404s

### PR DESCRIPTION
**Pull request by h3xar0n**
Resolves missing page errors from navigation links

### CODE REVIEW

File: app/enterprise/0.35-x/plugins/oidc-auth0.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+After creating your API, you will also need to add the `openid` scope at `https://manage.auth0.com/#/apis/<API ID>/scopes`.
+
+### Client Configuration
+
+You will need to authorize your client to access your API. Auth0 will prompt you to do so after client creation if you select the API you created previously from the client's Quick Start menu. After toggling the client to Authorized, expand its authorization settings and enable the `openid` scope.
+
+## Kong Configuration
+
+If you have not done so already, [create an API][create-api] to protect. The `url` configuration should match the Identifier you used when configuring Auth0.
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/plugins/oidc-auth0.md]:
```suggestion
If you have not done so already, [create a **Service**][add-service] to protect. The `url` configuration should match the Identifier you used when configuring Auth0.
```

File: app/enterprise/0.35-x/plugins/oidc-auth0.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+The service accessing your resource will need to pass its credentials to Kong. It can do so via HTTP basic authentication, query string arguments, or parameters in the request body. Basic authentication is generally preferable, as credentials in a query string will be present in Kong's access logs (and possibly in other infrastructure's access logs, if you have HTTP-aware infrastructure in front of Kong) and credentials in request bodies are limited to request methods that expect
+client payloads.
+
+For basic authentication, use your client ID as the username and your client secret as the password. For the other methods, pass them as parameters named `client_id` and `client_secret`, respectively.
+
+
+[client-credentials-grant]: https://auth0.com/docs/api-auth/tutorials/client-credentials
+[create-auth0-api]: https://auth0.com/docs/apis#how-to-configure-an-api-in-auth0
+[non-interactive-client]: https://auth0.com/docs/clients
+[create-api]: /0.13.x/admin-api/#add-api
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/plugins/oidc-auth0.md]:
```suggestion
[add-service]: /enterprise/{{page.kong_version}}/getting-started/add-service
```

File: app/enterprise/0.35-x/plugins/oidc-google.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+standardized][oidc-standard-claims], however--if a provider returns an `email` claim, the contents will always be an email address.
+
+This also requires that clients login using an account mapped to some consumer, which may not be desirable (e.g. you apply OpenID Connect to a service, but only use plugins requiring a consumer on some routes). To deal with this, you can set the `anonymous` parameter in your OIDC plugin configuration to the ID of a generic consumer, which will then be used for all authenticated users that cannot be mapped to some other consumer.
+
+
+[add-certificate]: /1.0.x/admin-api/#add-certificate
+[google-oidc]: https://developers.google.com/identity/protocols/OpenIDConnect
+[google-create-credentials]: https://console.developers.google.com/apis/credentials
+[add-service]: /enterprise/{{page.kong_version}}/getting-started/add-service/
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/plugins/oidc-google.md]:
```suggestion
[add-service]: /enterprise/{{page.kong_version}}/getting-started/add-service
```

**Comment by levfishbluefish** [comment]
Do we need any callouts for Sessions in these docs?

**Comment by h3xar0n** [comment]
@levfishbluefish negative, insofar as they don't mention enforcing RBAC / starting config, Sessions are already set by the time we get here.

**Pull request merged**